src/components/ControllerTop.vue

```diff
@@ -372,34 +375,6 @@ export default {
     },
     blur() {
       this.startListening();
-    },
-    logCompile(keyboard) {
-      this.$ga.event({
-        eventCategory: 'apicall',
-        eventAction: 'compilation',
-        eventLabel: keyboard
-      });
-    },
-    logLoadDefaultSuccess(keyboardName) {
-      this.$ga.event({
-        eventCategory: 'apicall',
-        eventAction: 'loadDefaultSuccess',
-        eventLabel: keyboardName
-      });
-    },
-    logLoadDefaultFail(keyboardName) {
-      this.$ga.event({
```

Comment:
Deleting this, we might delete ga.js?

Reply:
Eventually. When editing a file if you see it, we can start removing it. Don't want to blow up the pr.